CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 1.35.0 [unreleased]
 
+### Bug Fixes
+1. [#526](https://github.com/influxdata/influxdb-client-python/pull/526): Creating client instance from static configuration
```

Comment:
Is this PR in reference to #522?

Reply:
No, it is fix for https://github.com/influxdata/influxdb-client-python/issues/520